Dockerfile

```
@@ -12,10 +12,10 @@ RUN apt-get update -y && \
 
 WORKDIR $SRC_PATH
 
-RUN git clone https://github.com/filecoin-project/sentinel-archiver.git && \
-    cd sentinel-archiver && make build
+RUN git clone https://github.com/filecoin-project/lily-archiver.git && \
+    cd lily-archiver && make build
 
-RUN git clone --branch v0.13.0 https://github.com/filecoin-project/lily.git && \
+RUN git clone --branch frrist/fix-vm_messages-parsing https://github.com/filecoin-project/lily.git && \
```

I think we can use `main` now as the branch was merged. 

```suggestion
RUN git clone https://github.com/filecoin-project/lily.git && \
```